src/@context/Pool/index.tsx

```diff
@@ -119,11 +119,7 @@ function PoolProvider({ children }: { children: ReactNode }): ReactElement {
     if (!poolData) return
 
     // Total Liquidity
-    const totalLiquidityInOcean = isValidNumber(poolData.spotPrice)
-      ? new Decimal(poolData.baseTokenLiquidity).add(
-          new Decimal(poolData.datatokenLiquidity).mul(poolData.spotPrice)
-        )
-      : new Decimal(0)
+    const totalLiquidityInOcean = new Decimal(poolData.baseTokenLiquidity || 0)
```

Comment:
from UI standpoint this now makes sense but as said yesterday, based on this code it does not make sense. For whatever reason `poolData.baseTokenLiquidity` is already doubled, hence the issue with the OCEAN/DT further down